src/backend/joanie/urls.py

```diff
@@ -74,6 +74,16 @@
     api_client.OrganizationAccessViewSet,
     basename="organization_accesses",
 )
+organization_related_router.register(
+    "courseproductrelations",
```

Comment:
Just to stick to naming used in other routes.

```suggestion
    "course-product-relations",
```